server/utils/nunjucksSetup.ts

```diff
@@ -53,6 +53,11 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
 
   njkEnv.addFilter('toMonitorName', (val: string) => formatMonitorName(val))
 
+  njkEnv.addFilter(
```

Comment:
little util to help developing templates, this filter pretty prints the model in nested json